### PR TITLE
Project activity cards

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: src/reducers/posts.js [hunk 1]
```diff
@@ -67,7 +67,58 @@ const updatePostProps = (state, postId, props) => {
...
+    type: 'project',
+    community_ids: [
+      '29'
+    ],
+    voter_ids: [],
+    follower_ids: [
+      '11204'
+    ],
+    user_id: '11204',
+    newActivity: {
```
Comment (robbiecarlton) [inline src/reducers/posts.js]: @levity - how do you feel about this structure in the reducer.
We need the post and the comment, and users for both. Right now it's all denormalized. I thought about adding clauses to the post, comment and user reducers to pull out the nested stuff, but couldn't decide what was better (and this way is definitely easier ;) )

**Comment by robbiecarlton** [comment]
screen shot 2017-01-13 at 10 16 59 am
screen shot 2017-01-13 at 10 31 05 am

**Comment by robbiecarlton** [comment]
This depends on https://github.com/Hylozoic/hylo-node/pull/210

File: src/components/Post.js [hunk 2]
```diff
@@ -70,10 +71,14 @@ export class Post extends React.Component {
 
   render () {
     let { post } = this.props
-    const { comments, expanded, onExpand, community, dispatch, contributorChoices } = this.props
+    const { expanded, onExpand, community, dispatch, contributorChoices } = this.props
+    var comments = this.props.comments
```
Comment (levity) [inline src/components/Post.js]: you can put this in with `let { post }` on line 73

**Review by levity — APPROVED** [review]
other than that minor tweak i commented on, looks good

**Pull request merged**